[2023-03-21T11:30:31.744+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_python_operator_v05.install_odbc_1 scheduled__2023-03-19T00:00:00+00:00 [queued]>
[2023-03-21T11:30:31.764+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_python_operator_v05.install_odbc_1 scheduled__2023-03-19T00:00:00+00:00 [queued]>
[2023-03-21T11:30:31.764+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T11:30:31.765+0000] {taskinstance.py:1283} INFO - Starting attempt 2 of 6
[2023-03-21T11:30:31.766+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T11:30:31.790+0000] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): install_odbc_1> on 2023-03-19 00:00:00+00:00
[2023-03-21T11:30:31.798+0000] {standard_task_runner.py:55} INFO - Started process 7920 to run task
[2023-03-21T11:30:31.803+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_operator_v05', 'install_odbc_1', 'scheduled__2023-03-19T00:00:00+00:00', '--job-id', '282', '--raw', '--subdir', 'DAGS_FOLDER/python_operator.py', '--cfg-path', '/tmp/tmpdmsibkwx']
[2023-03-21T11:30:31.806+0000] {standard_task_runner.py:83} INFO - Job 282: Subtask install_odbc_1
[2023-03-21T11:30:31.892+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_with_python_operator_v05.install_odbc_1 scheduled__2023-03-19T00:00:00+00:00 [running]> on host cfacfd39f993
[2023-03-21T11:30:31.997+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mr.Phi
AIRFLOW_CTX_DAG_ID=dag_with_python_operator_v05
AIRFLOW_CTX_TASK_ID=install_odbc_1
AIRFLOW_CTX_EXECUTION_DATE=2023-03-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-19T00:00:00+00:00
[2023-03-21T11:30:31.998+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-21T11:30:32.000+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -']
[2023-03-21T11:30:32.011+0000] {subprocess.py:86} INFO - Output:
[2023-03-21T11:30:32.022+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2023-03-21T11:30:32.022+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2023-03-21T11:30:32.100+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
[2023-03-21T11:30:32.103+0000] {subprocess.py:93} INFO - E: This command can only be used by root.
[2023-03-21T11:30:32.379+0000] {subprocess.py:93} INFO - 100   983  100   983    0     0   2753      0 --:--:-- --:--:-- --:--:--  2753
[2023-03-21T11:30:32.380+0000] {subprocess.py:93} INFO - (23) Failed writing body
[2023-03-21T11:30:32.382+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-03-21T11:30:32.402+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-03-21T11:30:32.408+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_python_operator_v05, task_id=install_odbc_1, execution_date=20230319T000000, start_date=20230321T113031, end_date=20230321T113032
[2023-03-21T11:30:32.430+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 282 for task install_odbc_1 (Bash command failed. The command returned a non-zero exit code 1.; 7920)
[2023-03-21T11:30:32.460+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-03-21T11:30:32.492+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
